dpti/workflows/simulations/ti_sim.py

```
@@ -0,0 +1,689 @@
+#             # result_file_path = os.path.join(io_handler.job_dir, "result.json")
+#         return info
+
+
+# %%
+
+
+TIIntegrationPathType = TypeVar("TIIntegrationPathType", bound="TIIntegraionPath")
+
+
+class TIIntegraionPath:
+    _instances = {}
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)